.bazelrc

```diff
@@ -1 +1 @@
-build --enable_workspace
+# .bazelrc
```

Comment:
Just double checking that this shoould be empty. If so, we might want to change the comment to `# Intentionally blank` or just delete the file.

Reply:
I went with intentionally blank.
That way if we add things temporarily and remove things, we will have a history of that file, rather than disjoint appearancs and deletions.